index.html

```diff
@@ -14,23 +14,157 @@
     <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
     <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@100;200;300;400;500;600;700&display=swap"
         rel="stylesheet">
+    <!-- <link rel="stylesheet" media="screen and (min-width: 481px)" href="desktop.css"> -->
 </head>
 
 <body>
-    <section class="headline-container">
+    <section class="headline-main">
         <header class="header">
             <nav>
-                <a href="#"><span>My Logo</span></a>
-                <a href="#"><i class="fas fa-bars"></i></a>
+                <a href="#" class="logo"><span>My Logo</span></a>
+                <div class="menu">
+                    <a href="#" class="desktop-link">Portfolio</a>
+                    <a href="#" class="desktop-link">About</a>
+                    <a href="#" class="desktop-link">Contact</a>
+                </div>
+                <a href="#" class="mobile-link menu"><i class="fas fa-bars"></i></a>
             </nav>
         </header>
-    
+        
         <div class="headline">
```

Comment:
- [ ] Can you please add some space between the `header` and the `headline` sections as in the Figma design? see the difference below: 👇 
- Your design
![image](https://user-images.githubusercontent.com/55465133/130153147-fed19bdd-6f51-4c76-a0b9-e1a4b7037291.png)

- Figma design
![image](https://user-images.githubusercontent.com/55465133/130153185-cc2b490b-714c-444f-89ac-926cd1f534ae.png)

- [ ] [OPTIONAL] I think it could be nicer to use the `<section>` tag for the headline section instead of a `<div>` since it's a big section.